roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml

```diff
@@ -0,0 +1,114 @@
+---
+
+- name: Delete pre-existing heketi resources
+  oc_obj:
+    namespace: "{{ glusterfs_namespace }}"
+    kind: "{{ item.kind }}"
+    name: "{{ item.name | default(omit) }}"
+    selector: "{{ item.selector | default(omit) }}"
+    state: absent
+  with_items:
+  - kind: "template,route,service,dc,jobs,secret"
+    selector: "deploy-heketi"
+  - kind: "svc"
+    name: "heketi-storage-endpoints"
+  - kind: "svc"
+    name: "heketi-storage"
+  - kind: "secret"
+    name: "heketi-{{ glusterfs_name }}-topology-secret"
+  - kind: "template,route,service,dc"
+    name: "heketi-{{ glusterfs_name }}"
+  - kind: "svc"
+    name: "heketi-db-{{ glusterfs_name }}-endpoints"
+  - kind: "sa"
+    name: "heketi-{{ glusterfs_name }}-service-account"
+  - kind: "secret"
+    name: "heketi-{{ glusterfs_name }}-admin-secret"
+  - kind: "secret"
+    name: "heketi-{{ glusterfs_name }}-config-secret"
+  failed_when: False
+
+- name: Delete pre-existing GlusterFS resources
+  oc_obj:
+    namespace: "{{ glusterfs_namespace }}"
+    kind: "{{ item.kind }}"
+    name: "{{ item.name }}"
+    state: absent
+  with_items:
+  - kind: template
+    name: glusterfs
+  - kind: daemonset
+    name: "glusterfs-{{ glusterfs_name }}"
+  - kind: storageclass
+    name: "glusterfs-{{ glusterfs_name }}"
```

Comment:
We should also delete the glusterblock-provisioner pod. I believe that's part of a DeploymentConfig.

Reply:
I dont' see glusterblock provisioner starting up as part of default CNS setup.  Also, all the resources in glusterfs namespace gets deleted.

Reply:
Note that we'll need to cover the uninstall of gluster-s3, but I say we can do that later.

Reply:
sure - will check about the block part